src/routes/+layout.svelte

```
@@ -1,14 +1,45 @@
+		return parts.join('');
+	});
 </script>
 
 <svelte:head>
-	<link rel="icon" type="image/png" href={PUBLIC_OBA_LOGO_URL} />
-	<title>{PUBLIC_OBA_REGION_NAME}</title>
+	<link rel="icon" type="image/png" href={logoUrl} />
+	<title>{regionName}</title>
 	<meta name="viewport" content="width=device-width, initial-scale=1" />
+	{#if themeStyle}
+		{@html `<style>${themeStyle}</style>`}
```

ESLint

[error] `{`@html`}` can lead to XSS attack.

(svelte/no-at-html-tags)